src/index.html

```diff
@@ -20,8 +20,434 @@
+
+                <div class="cardPrice__price-2">$2,199</div>
+              </div>
+            </div>
+
+            <div class="button">
+              <a
+                class="button__element"
+                href="#"
+              >
+                BUY
+              </a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="cardTop">
+            <div class="cardTop__Img"></div>
+
+            <h1 class="cardTop__h1">
+              APPLE A1419 iMac 27" Retina
+              <br />
+              5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <h2 class="cardTop__h2">Product code: 195434</h2>
+          </div>
+
+          <div class="cardBottom">
+            <div class="starsReviews">
+              <div class="stars stars--5">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="reviews">Reviews: 5</div>
+            </div>
+
+            <div class="cardPrice-mix">
+              <div class="cardPrice">
+                <div class="cardPrice__price-1">Price:</div>
+
+                <div class="cardPrice__price-2">$2,199</div>
+              </div>
+            </div>
+
+            <div class="button">
+              <a
+                class="button__element"
+                href="#"
+              >
+                BUY
+              </a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="cardTop">
+            <div class="cardTop__Img"></div>
+
+            <h1 class="cardTop__h1">
+              APPLE A1419 iMac 27" Retina
+              <br />
+              5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <h2 class="cardTop__h2">Product code: 195434</h2>
+          </div>
+
+          <div class="cardBottom">
+            <div class="starsReviews">
+              <div class="stars stars--5">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="reviews">Reviews: 5</div>
+            </div>
+
+            <div class="cardPrice-mix">
+              <div class="cardPrice">
+                <div class="cardPrice__price-1">Price:</div>
+
+                <div class="cardPrice__price-2">$2,199</div>
+              </div>
+            </div>
+
+            <div class="button">
+              <a
+                class="button__element"
+                href="#"
+              >
+                BUY
+              </a>
+            </div>
+          </div>
+        </div>
```

Comment:
All product cards have identical content. If this is not intentional, consider using a dynamic approach to render product cards with varying content. This can be achieved using a templating engine or JavaScript to populate the cards with data.